### PR TITLE
feat(sdk-coin-avaxp): Add Verify for Import in C

### PR DESCRIPTION
**Pull request by colin-y-wu**
Ticket: BG-55596

## Description

Add verify functionality for ImportInC transactions

## Issue Number

BG-55596

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by jicli — DISMISSED** [review]

File: modules/sdk-coin-avaxp/src/avaxp.ts [hunk 1]
```diff
@@ -90,6 +92,32 @@ export class AvaxP extends BaseCoin {
           }
         }
         break;
+      case TransactionType.Import:
```
Comment (jicli) [inline modules/sdk-coin-avaxp/src/avaxp.ts:95]: this is not import in C, this is import from c

File: modules/sdk-coin-avaxp/src/avaxp.ts [hunk 2]
```diff
@@ -90,6 +92,32 @@ export class AvaxP extends BaseCoin {
...
+          const fee = new BigNumber(explainedTx.fee.fee);
+          const outputAmount = new BigNumber(explainedTx.outputs[0].amount);
+          const totalOutputAmount = new BigNumber(explainedTx.outputAmount);
+
+          if (!outputAmount.plus(fee).isEqualTo(params.txParams.recipients[0].amount)) {
+            throw new Error('C-Chain amount does not match with expected txParams');
+          }
+
+          if (!totalOutputAmount.plus(fee).isEqualTo(params.txParams.recipients[0].amount)) {
+            throw new Error('C-Chain output amount does not match with expected txParams');
```
Comment (jicli) [inline modules/sdk-coin-avaxp/src/avaxp.ts:117]: let's update the throw message as i mentioned in https://github.com/BitGo/bitgo-microservices/pull/25063 ty

File: modules/sdk-coin-avaxp/src/avaxp.ts (hunk 1 above)
Comment (colin-y-wu) [reply modules/sdk-coin-avaxp/src/avaxp.ts:99]: added check after this to confirm that we have import in c txn

**Review by OttoAllmendinger — DISMISSED** [review]
most of these seem to formatting changes unrelated to the PR description

**Comment by mmcshinsky-bitgo** [comment]
@colin-y-wu You'll need to rebase this PR to fix the failed test.

**Review by OttoAllmendinger — COMMENTED** [review]
nothing for me to review

File: modules/sdk-coin-avaxp/src/lib/transaction.ts [hunk 3]
```diff
@@ -349,4 +349,11 @@ export class Transaction extends BaseTransaction {
       memo: txJson.memo,
     };
   }
+
+  /**
+   * Check if this transaction is a P chain
+   */
+  get isTransactionForPChain(): boolean {
```
Comment (CapnMigraine) [inline modules/sdk-coin-avaxp/src/lib/transaction.ts:356]: I think import to P should be the default. Could we replace this function with one called `isImportToC`?

**Review by CapnMigraine — DISMISSED** [review]

**Review by jicli — DISMISSED** [review]

**Review by CapnMigraine — APPROVED** [review]

**Pull request merged**